documentation/docs/admin/accounts.md

```diff
@@ -26,7 +26,9 @@
 - `Team id`: Used to select the team this user is part of (can be blank)
 - `Team admin`: This should be checked if the user is the admin of the team above
 - `Reset Token`: Ignore
-- `Reset Time`: Sets challenge reset date and time (leave this as blank)
+- `Reset Time`: Ignore - Time the token should reset
+- `challenge_start_date`: Ignore - Sets challenge reset date and time (leave this as blank)
```

Comment:
I think the descriptions for lines 28-30 are all mixed around

Reply:
reset time is about the token - I dont think its currently used anywhere tbh

Reply:
Can you reword that to make it more clear? Such as "Sets a time for the token above to reset".
Also is challenge start date used to "set the challenge reset date and time" or "used to calculate the challenge reset date and time", the latter makes more sense.